client/components/dateForm.jsx

```diff
@@ -9,11 +10,14 @@ export default class DateForm extends React.Component {
     const selectedYear = new Date().getFullYear();
     const dayNames = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
     const monthNames = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
+    const activities = ['Eating', 'Shopping', 'Hiking', 'Picnic', 'Movies', 'Spa Day', 'Bowling', 'Other'];
```

Comment:
You can put this variables on line 5. There's no reason to put them in state. they never change. only put values that change in state.